docs/rose-suite-run-proposal/future-cli-conventions.md

```diff
@@ -1,14 +1,37 @@
-Placeholder - populate in new PR
 # `cylc flow` CLI proposal
-Refers to discussion in [cylc-flow #1030](https://github.com/cylc/cylc-flow/issues/1030).
 
+## Purpose
+Specify how the Cylc8 command to replace `rose-suite-run` will work.
+
+## References
+[cylc-flow #1030](https://github.com/cylc/cylc-flow/issues/1030).
 
 ## Discussion of possible names for new command
 
+Possibilities include:
+
+| Name | For | Against |
+| --- | --- | --- |
+|`cylc run`||Already in use for another command|
+|`cylc suite-run`|similar to `rose suite-run` -familiar|similar to `rose suite-run` -confusing<br>Quite long|
+|`cylc`|Short<br>Hints at the core-ness of this command to Cylc|Already in use|
+|`cylc flow`|Consistent naming<br>Like nothing currently in use  ||
```

Comment:
against: is a noun not a verb?

Reply:
To be against - surely a verb?

Reply:
Forgive me for being terrible, but:
 
![scr-define-flow](https://user-images.githubusercontent.com/30274190/62317003-a3541e00-b490-11e9-83f4-710e83419551.png)

Reply:
...either way, I personally think ``cylc flow`` is not a good idea. It's already the name of the module, & "flow" doesn't imply (at least to me) start-up, which is the general idea (which is probably what Oliver was getting at?).

I think we should choose something that is some synonym of 'start-up' or perhaps of 'install', or something that implies both if there is such a word out there. Something more like ``cylc go``, which is my favourite there, but I still am not that keen on. Suggestions I have at the moment along those lines are ``cylc ...``

* ``launch``;
* ``start``;
* ``activate``;
* ``advance``;
* ``progress``;

Another one on a different train of thought:

* ``work``, c.f. the new (generic) term workflow & the work directory, i.e. let my suite continue with its work), or similarly even ``workout``.

Could we perhaps get some suggestions along those lines, as you prefer, added into the list of possible names?

Reply:
Or `cylc play`. Play-Pause-Stop like a music player. (This is used by the old SMS.)

But I like `cylc go` a lot. Fewer letters to type.

Reply:
Yes, actually both of those are nice.

And there's the potential for puns involving suite names for the latter, whether that's a good or bad thing I am not sure (``cylc go away``, ... ``go fish``,... ``go to_sleep_for_goodness_sake_hilary_we_know_its_2am_in_NZ``...)

Reply:
Would be happy with play, pause, stop. Would want to symlink "run" to "play" though.

Reply:
The UM is probably something by Bach.

Reply:
@wxtim - can you add the above suggestions (maybe with args for and against) to the document, then we'll merge this and propose any further changes by subsequent PRs.